### ISSUE TITLE
tag messup

### ISSUE THREAD

**Issue by noXi89** [AntoineMary/grunt-angular-file-loader#10]
Auto-indentation is not working (I guess). It duplicated tags from above.

Template-File:

```

  <!-- build:js({.tmp,app}) scripts/scripts.js -->

  <!-- angular -->
  <!-- endangular -->

  <!-- endbuild -->

```

Result:

```

  <!-- build:js({.tmp,app}) scripts/scripts.js -->

  <!-- angular -->

  <!-- build:js({.tmp,app}) scripts/scripts.js -->

  <script src="modules/directives/...js" type="text/javascript"></script>

  <!-- build:js({.tmp,app}) scripts/scripts.js -->

  <!-- endangular -->

  <!-- endbuild -->

```

**Comment by noXi89** [comment]
**Example 2**

Temlpate:

```
<html>
<body>
  <!-- angular -->
  <!-- endangular -->
</body>
</html>
```

Result:

```
<html>
<body>
  <!-- angular -->
<html>
<body>
  <script src="app.js" type="text/javascript"></script>
<html>
<body>
  <!-- endangular -->
</body>
</html>
```

**Comment by noXi89** [comment]
fixed in #11 please review
